docs/user_guide/data_model.rst

```diff
@@ -28,28 +28,44 @@ The table below summarizes mandatory fields used in all parameter definitions.
 
    * - **type**
      - Parameter type
-     - file, dir, str, float, bool, int, [], (a,b)
+     - file, dir, str, float, bool, int, [], enum, tuple
+
+   * - **enum**
+     - List of legal strings for enum type
+     - List of Strings
+
+   * - **unit**
+     - Implied unit for parameter value
+     - String
 
    * - **defvalue**
      - Default schema value
      - Type dependent
 
-   * - **shorthelp**
-     - Short single line help string.
-     - String
+   * - **value**
+     - Global schema value
+     - Type dependent
 
-   * - **help**
-     - Multi-line documentation string
-     - String
+   * - **nodevalue**
+     - Dictionary of values based on step & index keys
+     - Dictionary
 
-   * - **example**
-     - Usage examples for CLI ad API
-     - String
+   * - **pernode**
+     - Enables/disables setting of value on a per node basis
+     - never, required, optional
```

Comment:
maybe quotes around these to indicate they are strings (consistent with "scope" listing)